src/Symfony/Component/Console/Helper/TableStyle.php

```diff
@@ -108,18 +117,71 @@ public function getVerticalBorderChar()
         return $this->verticalBorderChar;
     }
 
+    /**
+     * Sets crossing characters.
+     *
+     * Example:
+     * <code>
+     * 1---------------2-----------------------2------------------3
+     * | ISBN          | Title                 | Author           |
+     * 8---------------0-----------------------0------------------4
+     * | 99921-58-10-7 | Divine Comedy         | Dante Alighieri  |
+     * | 9971-5-0210-0 | A Tale of Two Cities  | Charles Dickens  |
+     * | 960-425-059-0 | The Lord of the Rings | J. R. R. Tolkien |
+     * 7---------------6-----------------------6------------------5
+     * </code>
+     *
+     * @param string $cross       Crossing char (see #0 of example)
+     * @param string $topLeft     Top left char (see #1 of example)
+     * @param string $topMid      Top mid char (see #2 of example)
+     * @param string $topRight    Top right char (see #3 of example)
+     * @param string $midRight    Mid right char (see #4 of example)
+     * @param string $bottomRight Bottom right char (see #5 of example)
+     * @param string $bottomMid   Bottom mid char (see #6 of example)
+     * @param string $bottomLeft  Bottom left char (see #7 of example)
+     * @param string $midLeft     Mid left char (see #8 of example)
+     *
+     * @return $this
+     */
+    public function setCrossingChars(string $cross, string $topLeft, string $topMid, string $topRight, string $midRight, string $bottomRight, string $bottomMid, string $bottomLeft, string $midLeft): self
+    {
+        $this->crossingChar = $cross;
+        $this->crossingTopLeftChar = $topLeft;
+        $this->crossingTopMidChar = $topMid;
+        $this->crossingTopRightChar = $topRight;
+        $this->crossingMidRightChar = $midRight;
+        $this->crossingBottomRightChar = $bottomRight;
+        $this->crossingBottomMidChar = $bottomMid;
+        $this->crossingBottomLeftChar = $bottomLeft;
+        $this->crossingMidLeftChar = $midLeft;
+
+        return $this;
+    }
+
+    /**
+     * Sets default crossing character used for each cross.
+     *
+     * @see {@link setCrossingChars()} for setting each crossing individually.
+     */
+    public function setDefaultCrossingChar(string $char): self
+    {
+        return $this->setCrossingChars($char, $char, $char, $char, $char, $char, $char, $char, $char);
+    }
+
     /**
      * Sets crossing character.
      *
      * @param string $crossingChar
      *
      * @return $this
+     *
+     * @deprecated since version 4.1, to be removed in 5.0. Use {@link setDefaultCrossingChar()} instead.
```

Comment:
`to be removed in Symfony 5.0`

Reply:
Actually, we settled on the contrary. since Symfony 4.1, to be removed in 5.0.

Reply:
Done